### ISSUE TITLE
Use existing Vocabulary terms from ActivityStreams for Link semantics

### ISSUE THREAD

**Issue by BigBlueHat** [w3c/pub-manifest#14]
Given that ActivityStreams is widely used (thanks to Mastadon, PeerTube, etc), it would make sense for our Link definitions to build up from theirs--at least in the places where Schema.org's vocabulary is lacking:
https://www.w3.org/TR/activitystreams-vocabulary/#dfn-link

The [Web of Things spec](https://www.w3.org/TR/wot-thing-description/#link) also has a nearly identical definition and hopefully they could be convinced to match the ActivityStreams terminology also.

| WPUB | ActivityStreams | Web of Things |
|--|--|--|
| url | href* | href |
| encodingFormat | mediaType | mediaType |
| name | name | |
| description | | |
| rel | rel | rel |
|  | hreflang |  |
|  | height |  |
|  | width |  |
|  | | anchor |
|  | preview |  |

* ActivityStreams also defines `url` which has a range of `xsd:anyURI` (i.e. a "raw" URL in the form of `http://example.com/`) OR a [`Link` object](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-link) whereas their [`href`](https://www.w3.org/TR/activitystreams-vocabulary/#dfn-href) is defined with a range of *only* `xsd:anyURI`

At the very least, it would be helpful if the three communities could coordinate on their use of `rel` and `mediaType`. I've also found the distinction between `url` and `href` to be valuable.

Context files for each community:
 - https://www.w3.org/ns/wp-context
 - https://www.w3.org/ns/activitystreams.jsonld
 - https://www.w3.org/ns/td.jsonld

**Comment by iherman** [comment]
I wonder whether this issue is still relevant (it has not be commented upon for more than a year...)

**Comment by BigBlueHat** [comment]
> Naive question: why didn't schema.org use such common terms as `mediaType` and `href`?

Schema.org seems to have been more liberal with their definition of `encodingFormat`:
> Unregistered or niche encoding and file formats can be indicated instead via the most appropriate URL, e.g. defining Web page or a Wikipedia/Wikidata entry.

In many ways its analogous to `dcterms:format` http://purl.org/dc/terms/format which is frequently used for a MIME media type...but not always.

Not sure on the `url` vs. `href` comparison, however.

**Comment by BigBlueHat** [comment]
> I wonder whether this issue is still relevant (it has not be commented upon for more than a year...)

Mostly, I was wanting us to attempt some sort of cross-compatibility with other W3C-based specs. For starters ActivityStreams and Web of Things both have a `rel` value--which Schema.org currently lacks. So, if we add that, and upstream it to Schema.org, we should (ideally) build off of their definitions.

Given that we are primarily focused on Schema.org use, we should do more analysis of when we're using what and how. For instance, the `LinkedResource` class was created for WPUB, but `MediaObject` from Schema.org is painfully close. If Schema.org compatibility/value-benefits are still driving our choices, then we should be clear about why we've chosen to not use `MediaObject` (and `contentUrl`, etc.) or reconsider that decision.

**Comment by iherman** [comment]
I think trying to get close to schema.org is obviously important. So far we tried and failed...

- `MediaObject` would not be a good alternative imho. It is a subtype of `CreativeWork`, and I would not think a `LinkedResource` should be restricted to creative works (it could be used for  a metadata, for example...)
- The really good target for a connection would have been `LinkedRole` but that missed some necessary entries. See https://github.com/schemaorg/schemaorg/issues/1045 where we tried to see if there is a way to bring these close, but we did not succeed. At a few weeks away from a possible CR I am not sure we have a real way forward here.
